process/jira.md

```diff
@@ -0,0 +1,54 @@
+-   Validating each ticket still makes sense and is correctly prioritized as in agile development priorities might shift wildly. In case of failure to pass this test ticket should be deprioritized or even closed with a clear comment why. Consider commenting even when deprioritizing to keep - potentially unknown - colleagues and stakeholders in the loop and able to react.
+-   Making sure unpopular/maintenance/forgotten tickets are taken care of eventually.
+-   Validating all tickets have proper status:
+    -   Are in-progress tickets progressing? (communicating by putting ticket back from in progress to previous states is valid fix)
+    -   Are blocked tickets still blocked?
+    -   Are PRs being done in a timely manner?
+-   Is someone struggling with a ticket without a reaction?
+-   Validating miscellaneous conditions like are in-progress tickets assigned, all stakeholders are kept in the loop, etc.
+-   Ticket updates/comments are also clear and understandable. Questions are answered.
+-   Giving the team a good pat on the back for its work.
+
+## Responsibilities and Perks
+
+Main responsibility to make this happen is on [Lead Engineer](https://culture.mayadata.io/job-roles/lead-engineer.html) (put your [project manager](https://culture.mayadata.io/job-roles/lead-engineer.html#project-management)'s hat on), however she/he might choose to delegate or rotate responsibilities in the team. Some of the responsibilities might also be automated. Until they are, they however must be done manually. [Lead Engineer](https://culture.mayadata.io/job-roles/lead-engineer.html) should motivate the team to keep its Jira as clean as possible by default.
+
+It is recommended that *loop two* be done by [Lead Engineer](https://culture.mayadata.io/job-roles/lead-engineer.html) herself/himself before the team\'s regular grooming/prioritization sessions. It will have additional value of coming to grooming/prioritization meetings prepared with refreshed knowledge of tasks at hand as [Lead Engineer](https://culture.mayadata.io/job-roles/lead-engineer.html) often has to explain work at hand in terms of why and how it connects to other tasks in near term, other projects and motivate team members to do the work by conveying its purpose and meaning in a broader context. Moreover regular maintenance of workboard brings intuitive understanding of team's current priorities and short/mid-term plans which, from experience, is quite useful for leadership role and helps with many responsibilities [Lead Engineer](https://culture.mayadata.io/job-roles/lead-engineer.html) has as reporting progress outside the team (if at all needed with clean project), answering and aligning team members about priorities, avoiding work duplication and in general keeping the team "well oiled" and working at its best performance. And it feels really good.
```

Comment:
```suggestion
It is recommended that *the second loop* be done by [Lead Engineer](https://culture.mayadata.io/job-roles/lead-engineer.html) herself/himself before the team\'s regular grooming/prioritization sessions. It will have additional value of coming to grooming/prioritization meetings prepared with refreshed knowledge of tasks at hand as [Lead Engineer](https://culture.mayadata.io/job-roles/lead-engineer.html) often has to explain work at hand in terms of why and how it connects to other tasks in near term, other projects and motivate team members to do the work by conveying its purpose and meaning in a broader context. Moreover regular maintenance of workboard brings intuitive understanding of team's current priorities and short/mid-term plans which, from experience, is quite useful for leadership role and helps with many responsibilities [Lead Engineer](https://culture.mayadata.io/job-roles/lead-engineer.html) has as reporting progress outside the team (if at all needed with clean project), answering and aligning team members about priorities, avoiding work duplication and in general keeping the team "well oiled" and working at its best performance. And it feels really good.
```